docs_src/tutorial_1_hello_world.py

```diff
@@ -0,0 +1,20 @@
+from corva import Api, Corva, Event, State
+
+app = Corva()  # 1 initialize the app
+
+
+@app.stream  # 2 add decorator with needed event type to your function
+def user_job(event: Event, api: Api, state: State):
+    # 3 add parameters with predefined types, that will be injected automatically
```

Comment:
This line isn't clear that it's referring to the above parameters

Reply:
expanded the comment